integration/spark2/src/main/scala/org/apache/spark/sql/hive/CarbonFileMetastore.scala

```diff
@@ -286,12 +286,16 @@ class CarbonFileMetastore extends CarbonMetaStore {
       newTableIdentifier: CarbonTableIdentifier,
       oldTableIdentifier: CarbonTableIdentifier,
       thriftTableInfo: org.apache.carbondata.format.TableInfo,
-      schemaEvolutionEntry: SchemaEvolutionEntry,
+      schemaEvolutionEntry: List[SchemaEvolutionEntry],
```

Comment:
schemaEvolutionEntries or schemaEvolutionEntryList is better

Reply:
yes, can change to schemaEvolutionEntryList